.pre-commit-config.yaml

```
@@ -30,12 +30,12 @@ repos:
     - python
 
 - repo: https://github.com/psf/black
-  rev: 21.12b0
+  rev: 22.3.0
```

```suggestion
  rev: 21.12.b0
```
Not updating black just yet, as discussed in #1503.